src/components/Avatars/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+import {IAvatars} from './Avatars.types';
+
+const Avatars: React.FC<IAvatars> = ({users, isPrivate}) => {
+  if (isPrivate) {
+    const user = users?.[0];
+    return <Avatar src={user?.picture} alt={user?.username} />;
+  } else {
+    return (
+      <AvatarsContainer>
+        {users?.slice(0, 4).map((user, index) => (
+          <Avatar key={user.id} src={user.picture} alt={user.username} />
+        ))}
+      </AvatarsContainer>
+    );
+  }
+};
```

Comment:
isPrivate이 어떤의미인가요?
isPrivate플래그에 따라 다른 ui를 렌더하는 경우라면 렌더하는 부분에서 갈라주면 될 것 같아요.

컴포넌트의 렌더/ 로직 등 정의하는 공간을 구분하는 것도 맥락 파악에 용이하기도 하고요~


```suggestion
const Avatars: React.FC<IAvatars> = ({users, isPrivate}) => {
  const user = useMemo(() => users?.[0], [users]) // 요럴때 메모이제이션을 하는 거에요!

    return (
       isPrivate ? <Avatar src={user?.picture} alt={user?.username}/>
            : <AvatarsContainer>
                {users?.slice(0, 4).map((user, index) => (
                    <Avatar key={user.id} src={user.picture} alt={user.username}/>
                ))}
            </AvatarsContainer>
     )
 }
```